src/handlers/AmazonHandler.ts

```diff
@@ -0,0 +1,29 @@
+import { Handler, Job } from "../types";
+import axios from 'axios';
+
+const url = 'https://www.amazon.jobs/en/search.json?radius=24km&facets%5B%5D=normalized_country_code&facets%5B%5D=normalized_state_name&facets%5B%5D=normalized_city_name&facets%5B%5D=location&facets%5B%5D=business_category&facets%5B%5D=category&facets%5B%5D=schedule_type_id&facets%5B%5D=employee_class&facets%5B%5D=normalized_location&facets%5B%5D=job_function_id&facets%5B%5D=is_manager&facets%5B%5D=is_intern&offset=0&result_limit=10&sort=relevant&latitude=&longitude=&loc_group_id=&loc_query=India&base_query=Software%20Development&city=&country=IND&region=&county=&query_options=&';
```

Comment:
`result_limit=10`
we're only getting 10 jobs, can you add pagination or just increase limit if that works.
I checked till limit of 100, and that works, but obviously, 100 jobs are not all relevant, please add some more filtering to it. This is the reason I didn't do it myself 

While post processing should remove plenty of these, maybe adding some time filtering to not consider job openings opened before 1 month can help.

Reply:
- Sure, I checked the page, and there is no option for filtering on the API level based on the date 
- So, I am adding a filter after fetching the results!